sdk/tests/deqp/functional/gles3/es3fTextureFormatTests.js

```diff
@@ -176,7 +176,7 @@ es3fTextureFormatTests.Texture2DFormatCase.prototype.iterate = function() {
     // Compare and log.
     var isOk = glsTextureTestUtil.compareImages(referenceFrame, renderedFrame, threshold);
 
-    assertMsgOptions(isOk, es3fTextureFormatTests.testDescription(), true, true);
+    assertMsgOptions(isOk, es3fTextureFormatTests.testDescription(), true, false);
```

Comment:
Function assertMsgOptions() is defined here:
https://github.com/KhronosGroup/WebGL/blob/master/sdk/tests/js/js-test-pre.js#L679
If test fails, assertMsgOptions() will call testFailedOptions():
https://github.com/KhronosGroup/WebGL/blob/master/sdk/tests/js/js-test-pre.js#L311
If exthrow is true, then testFailedOptions() will throw an exception and destroy current function stack.
For example, this.m_curFace will not be updated when an exception is thrown in this.testFace():
https://github.com/KhronosGroup/WebGL/blob/master/sdk/tests/deqp/functional/gles3/es3fTextureFormatTests.js#L328